device/accton/x86_64-accton_csp7551-r0/Accton-CSP7551/port_config.ini

```diff
@@ -0,0 +1,58 @@
+# name          lanes              speed     fec    autoneg   index   alias
+Ethernet1       0,1,2,3            100000    rs     2         1       Ethernet1   
```

Comment:
I believe the right value for `autoneg` will be `on`/`off`
https://github.com/Azure/sonic-swss/blob/f4ec565b44905fc8dd4d932a719b8425c8e5ea17/cfgmgr/buffermgrdyn.cpp#L2141